tests/turnt.toml

```diff
@@ -170,3 +170,21 @@ output.gfa = "-"
 [envs.flatgfa_file_inplace]
 command = "../flatgfa/target/debug/fgfa -m -p 128 -o {base}.inplace.flatgfa -I {filename} ; ../flatgfa/target/debug/fgfa -m -i {base}.inplace.flatgfa"
 output.gfa = "-"
+
+[envs.odgi_depth]
+binary = true
+command = "odgi depth -d -i {filename}"
+output.depth = "-"
+
+[envs.flatgfa_depth]
+command = "../flatgfa/target/debug/fgfa -I {filename} depth"
+output.depth = "-"
+
+[envs.chop_oracle_fgfa]
```

Comment:
Maybe this should be called `chop_oracle_with_links` or something to describe it somewhat more specifically?